src/entities/User/types.ts

```diff
@@ -0,0 +1,16 @@
+export type UserAttributes = {
+  address: string
+  forum_id: number
+  verification_date: string
+}
+
+export type ValidationMessage = {
+  address: string
+  timestamp: string
+  messageTimeout: NodeJS.Timeout
```

Comment:
It's not entirely clear to me which case we should use here.. I think `message_timeout`